### ISSUE TITLE
Ability to delete/remove actual group chats and secret chats

### ISSUE THREAD

**Issue by jonnywilliamson** [vysheng/tg#524]
I'm just after writing a Laravel package as a wrapper around telegram-cli and during the process was testing out most of the features.

The one glaring command that seems to be missing is the ability to actually REMOVE or delete a group chat (and secret chat) from the cli.

We can create them
```
create_group_chat
create_secret_chat
```

We can add and remove people from them
```
chat_add_user 
chat_del_user 
```

But no ability to remove old ones once everyone else has left one!

Have I just missed something or am I doing it wrong?

**Comment by jonnywilliamson** [comment]
I also see that issue 
https://github.com/vysheng/tg/issues/343
https://github.com/vysheng/tg/issues/283
https://github.com/vysheng/tg/issues/496

Are all practically the same. Just wondering what your thoughts are @vysheng ?

**Comment by LukeLR** [comment]
I think a workaround could be to log in to webogram, and use that cli-client for chat deletion. If you don't need to do so very often, this would be possible, I think. Although I agree that it would be nice to have a feature to do that in tg-cli

**Comment by jonnywilliamson** [comment]
@LukeLR Yeah thats exactly how I'm doing it at the moment. To be  fair I have very few conversations and it's not a major issue for me, but I was curious why it was missing from the CLI. 

Just wondered if it was an oversight or indeed a deliberate omission.

**Comment by LukeLR** [comment]
Yeah, I agree, would be nice to know why it's missing ;) I guess that vysheng just didn't had the time for it yet ;)

**Comment by zyberspace** [comment]
According to the [telegram-api](https://core.telegram.org/methods#working-with-chats) there is actually no option to delete a group chat.

If you for example create a second chat with the same name, telegram just appends a "`#1`" to the original name.
So if "Test_Chat" already exists, the `print_name` will be "Test_Chat#1" while the title remains "Test&nbsp;Chat".

Maybe someone could ask the telgram-makers if there will ever be an option to delete a group-chat?
